core/domain/question_domain.py

```diff
@@ -236,7 +236,7 @@ def create_default_question(cls, question_id):
             Question. A Question domain object with default values.
         """
         default_question_state_data = exp_domain.State.create_default_state(
-            feconf.DEFAULT_INIT_STATE_NAME, is_initial_state=True)
+            None, is_initial_state=True)
```

Comment:
Ah, actually, maybe we should have a create_default_question_state() instead.

How do we handle the difference (on-creation) between the question default state destination being None (and normal state destinations being non-None)?

Reply:
Done, added create_default_question_state()